Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -692,24 +701,62 @@ public void refreshControls()
     if (mPreferences.getBoolean("showInputOverlay", true))
     {
       // Add all the enabled overlay items back to the HashSet.
-      if (EmulationActivity.isGameCubeGame() || mPreferences.getInt("wiiController", 3) == 0)
-      {
-        addGameCubeOverlayControls(orientation);
-      }
-      else if (mPreferences.getInt("wiiController", 3) == 4)
+      if (EmulationActivity.isGameCubeGame())
       {
-        addClassicOverlayControls(orientation);
+        switch (NativeLibrary
+                .GetConfig(SettingsFile.FILE_NAME_DOLPHIN + ".ini", Settings.SECTION_INI_CORE,
+                        SettingsFile.KEY_GCPAD_PLAYER_1, DISABLED_GAMECUBE_CONTROLLER))
```

Comment:
To match the C++ code, the default should be `EMULATED_GAMECUBE_CONTROLLER`. (For the reference, this only applies to the first player. The other players do indeed default to `DISABLED_GAMECUBE_CONTROLLER`.)